src/app/background/index.js

```diff
@@ -20,10 +20,17 @@ class ImagePrediction {
       const imageAltText = await this.getPrediction(image)
       this.images[req.url] = imageAltText
 
+      console.log(imageAltText)
       if (imageAltText) {
         const payload = {
           imageUrl: req.url,
-          imageAltText: imageAltText
+          imageAltText: `Soapie sees ${imageAltText}`
+        }
+        this.sendMessage(req.tabId, payload)
+      }else{
+        const payload = {
+          imageUrl: req.url,
+          imageAltText: `Soapie sees nothing.`
```

Comment:
```
imageAltText: `Soapie sees ${imageAltText ? imageAltText : "nothing"}`
```
so no need to copy over the whole section?

Reply:
or 
```
imageAltText: `Soapie sees ${imageAltText || "nothing"}`
```
might also work

Reply:
cool, still not get used to those js short hands but I know what you mean. Will update in a bit

Reply:
Done! If no more request I will merge this PR